src/docfx/lib/git/CommitBuildTimeProvider.cs

```diff
@@ -9,23 +9,16 @@ internal class CommitBuildTimeProvider
     private readonly Repository _repo;
     private readonly Config _config;
     private readonly string _commitBuildTimePath;
-    private readonly IReadOnlyDictionary<string, DateTime> _buildTimeByCommit;
+    private IReadOnlyDictionary<string, DateTime> _buildTimeByCommit;
 
     public CommitBuildTimeProvider(Config config, Repository repo)
     {
         _repo = repo;
         _config = config;
         _commitBuildTimePath = AppData.BuildHistoryStatePath;
-        _buildTime = config.BuildTime ?? DateTime.UtcNow;
+        _buildTime = config.BuildTime;
```

Comment:
Does it fix the problem if we simply promote the now fallback to a static variable?
```suggestion
        _buildTime = config.BuildTime ?? s_now;
```